tests/integration/telemetry/stackdriver/stackdriver_filter_audit_test.go

```diff
@@ -0,0 +1,112 @@
+// +build integ
+// Copyright Istio Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stackdriver
+
+import (
+	"testing"
+	"time"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/echo"
+	"istio.io/istio/pkg/test/scopes"
+	"istio.io/istio/pkg/test/util/file"
+	"istio.io/istio/pkg/test/util/retry"
+	"istio.io/istio/pkg/test/util/tmpl"
+)
+
+// TestStackdriverAuditLogging testing Authz Policy can config stackdriver with audit policy
+func TestStackdriverAuditLogging(t *testing.T) {
+	framework.NewTest(t).
+		Run(func(ctx framework.TestContext) {
+			sendTrafficForAudit(t)
+
+			for _, cl := range ctx.Clusters() {
+				scopes.Framework.Infof("Validating Audit policy and Telemetry for Cluster %v", cl)
+
+				args := map[string]string{
+					"Namespace": getEchoNamespaceInstance().Name(),
+				}
+				policies := tmpl.EvaluateAllOrFail(t, args, file.AsStringOrFail(t, "testdata/v1beta1-audit-authorization-policy.yaml.tmpl"))
+				ctx.Config().ApplyYAMLOrFail(t, "", policies...)
+				defer ctx.Config().DeleteYAMLOrFail(t, "", policies...)
+
+				retry.UntilSuccessOrFail(t, func() error {
+					clName := cl.Name()
+					if err := validateMetrics(t, serverRequestCount, clientRequestCount, clName); err != nil {
+						return err
+					}
+					t.Logf("Metrics validated for cluster %v", clName)
+					if err := validateLogs(t, serverLogEntry, clName); err != nil {
```

Comment:
Should add validation for audit log.. this will validate generic server access log